java/core/src/main/java/com/whylogs/core/metrics/MetricConfig.java

```diff
@@ -0,0 +1,52 @@
+package com.whylogs.core.metrics;
+
+import lombok.experimental.UtilityClass;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+@UtilityClass
+public class MetricConfig {
+    private final int hll_lg_k = 12;
```

Comment:
Can we override these defaults?

Reply:
The python had this as a frozen class so i thought these are final values, but if this should be changable that's fine to. As we are using this are these values that should stay constant?

Reply:
In Python the defaults can be overridden at construction time. They "can't" be changed after construction because they're frozen (I think the dictionary members are still mutable?)

Reply:
Ahh so our use case for this is that we may want this config overridden at construction but then frozen. Got it